[2025-04-22T23:44:57.912+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-22T23:44:57.926+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_data_etl_dbt.dbt_run_transformations scheduled__2025-04-21T03:00:00+00:00 [queued]>
[2025-04-22T23:44:57.932+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_data_etl_dbt.dbt_run_transformations scheduled__2025-04-21T03:00:00+00:00 [queued]>
[2025-04-22T23:44:57.932+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-22T23:44:57.940+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): dbt_run_transformations> on 2025-04-21 03:00:00+00:00
[2025-04-22T23:44:57.947+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=726) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-22T23:44:57.950+0000] {standard_task_runner.py:72} INFO - Started process 729 to run task
[2025-04-22T23:44:57.947+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'stock_data_etl_dbt', 'dbt_run_transformations', 'scheduled__2025-04-21T03:00:00+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/Stock_built_ELT.py', '--cfg-path', '/tmp/tmp8rro1q4q']
[2025-04-22T23:44:57.952+0000] {standard_task_runner.py:105} INFO - Job 50: Subtask dbt_run_transformations
[2025-04-22T23:44:57.986+0000] {task_command.py:467} INFO - Running <TaskInstance: stock_data_etl_dbt.dbt_run_transformations scheduled__2025-04-21T03:00:00+00:00 [running]> on host 712e28762a67
[2025-04-22T23:44:58.041+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='stock_data_etl_dbt' AIRFLOW_CTX_TASK_ID='dbt_run_transformations' AIRFLOW_CTX_EXECUTION_DATE='2025-04-21T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-21T03:00:00+00:00'
[2025-04-22T23:44:58.042+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-22T23:44:58.044+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-22T23:44:58.045+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt run --select models/intermediate/moving_averages models/intermediate/rsi --profiles-dir /opt/***/dbt --project-dir /opt/***/dbt']
[2025-04-22T23:44:58.051+0000] {subprocess.py:86} INFO - Output:
[2025-04-22T23:45:00.811+0000] {subprocess.py:93} INFO - [0m23:45:00  Running with dbt=1.7.19
[2025-04-22T23:45:01.721+0000] {subprocess.py:93} INFO - [0m23:45:01  Registered adapter: snowflake=1.7.1
[2025-04-22T23:45:01.741+0000] {subprocess.py:93} INFO - [0m23:45:01  Unable to do partial parsing because of a version mismatch
[2025-04-22T23:45:02.690+0000] {subprocess.py:93} INFO - [0m23:45:02  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-04-22T23:45:02.691+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-04-22T23:45:02.691+0000] {subprocess.py:93} INFO - - models.stock_data_transform_dbt.marts
[2025-04-22T23:45:02.714+0000] {subprocess.py:93} INFO - [0m23:45:02  Found 4 models, 1 snapshot, 1 source, 0 exposures, 0 metrics, 430 macros, 0 groups, 0 semantic models
[2025-04-22T23:45:02.717+0000] {subprocess.py:93} INFO - [0m23:45:02  The selection criterion 'models/intermediate/moving_averages' does not match any nodes
[2025-04-22T23:45:02.719+0000] {subprocess.py:93} INFO - [0m23:45:02  The selection criterion 'models/intermediate/rsi' does not match any nodes
[2025-04-22T23:45:02.720+0000] {subprocess.py:93} INFO - [0m23:45:02
[2025-04-22T23:45:02.721+0000] {subprocess.py:93} INFO - [0m23:45:02  Nothing to do. Try checking your model configs and model specification args
[2025-04-22T23:45:03.569+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-22T23:45:03.616+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-22T23:45:03.617+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=stock_data_etl_dbt, task_id=dbt_run_transformations, run_id=scheduled__2025-04-21T03:00:00+00:00, execution_date=20250421T030000, start_date=20250422T234457, end_date=20250422T234503
[2025-04-22T23:45:03.649+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-22T23:45:03.679+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-22T23:45:03.681+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
